leaddbsinterface/lead_settings.py

```diff
@@ -226,6 +230,34 @@ def make_oss_settings(self, hemis_idx: int = 0, output_path: str = "") -> dict:
             },
         }
 
+        # the constant model from Lead-DBS also assumes homogeneity
+        # but this can be customized
+        if cond_model == "Constant":
+            partial_dict["DielectricModel"]["CustomParameters"] = {
+                "Gray matter": {
+                    "permittivity": 2.22e4,
```

Comment:
Where do the default parameters come from?
Would be great to add a comment, because the permittivity seems low (is it from the kHz range?)

Reply:
this is just from GM at 10 kHz